app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_person = [Person(person["name"],
+                             person["age"]) for person in people]
+    for i, person in enumerate(people):
+        if person.get("husband"):
+            husband_name = person["husband"]
```

Comment:
You could use a walrus operator to not perform the operation of retrieving name twice during the check and variable assignment:
```suggestion
        if husband_name := person.get("husband"):
```